docs/resources/file.md.erb

```diff
@@ -87,6 +87,22 @@ The `mode` property tests if the mode assigned to the file matches the specified
 
     its('mode') { should cmp '0644' }
 
+You must write the numeric mode specification starting with zero, to tell Ruby
```

Comment:
Would you mind changing `Ruby` to `InSpec`? We have a lot of non-tech people using these docs and it may be more clear.